Scanning images: 100%|████████████████████| 5940/5940 [00:01<00:00, 5843.96it/s]
Scanning labels PCB-upright-28/train/labels.cache3 (1530 found, 0 missing, 4410 empty, 0 duplicate, for 5940 images): 5940it [00:00, 32173.76it/s]
Scanning images: 100%|██████████████████████| 576/576 [00:00<00:00, 3302.73it/s]
Scanning labels PCB-upright-28/valid/labels.cache3 (205 found, 0 missing, 371 empty, 0 duplicate, for 576 images): 576it [00:00, 13366.89it/s]
Images sizes do not match. This will causes images to be display incorrectly in the UI.
Image sizes 640 train, 640 test
Using 4 dataloader workers
Logging results to runs/train/yolor_p670
Starting training for 1000 epochs...
     Epoch   gpu_mem       box       obj       cls     total   targets  img_size
  0%|                                                   | 0/248 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "train.py", line 637, in <module>
    train(hyp, opt, device, tb_writer, wandb)
  File "train.py", line 346, in train
    pred = model(imgs)  # forward
  File "/scratch/anaconda3/envs/wvgPCB/lib/python3.7/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/scratch/WVG/yolor/models/models.py", line 543, in forward
    return self.forward_once(x)
  File "/scratch/WVG/yolor/models/models.py", line 604, in forward_once
    x = module(x)
  File "/scratch/anaconda3/envs/wvgPCB/lib/python3.7/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/scratch/anaconda3/envs/wvgPCB/lib/python3.7/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/scratch/anaconda3/envs/wvgPCB/lib/python3.7/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/scratch/anaconda3/envs/wvgPCB/lib/python3.7/site-packages/torch/nn/modules/batchnorm.py", line 136, in forward
    self.weight, self.bias, bn_training, exponential_average_factor, self.eps)
  File "/scratch/anaconda3/envs/wvgPCB/lib/python3.7/site-packages/torch/nn/functional.py", line 2058, in batch_norm
    training, momentum, eps, torch.backends.cudnn.enabled
RuntimeError: CUDA out of memory. Tried to allocate 30.00 MiB (GPU 0; 7.79 GiB total capacity; 6.42 GiB already allocated; 29.31 MiB free; 6.48 GiB reserved in total by PyTorch)